drivers/serial/uart_mcux_lpuart.c

```diff
@@ -970,9 +974,47 @@ static void mcux_lpuart_isr(const struct device *dev)
 }
 #endif /* CONFIG_UART_MCUX_LPUART_ISR_SUPPORT */
 
+#if LPUART_HAS_MODEM
+static int mcux_lpuart_config_flowctrl(uint8_t flow_ctrl, lpuart_config_t *uart_config)
+{
+	switch (flow_ctrl) {
+	case UART_CFG_FLOW_CTRL_NONE:
+		uart_config->enableTxCTS = false;
+		uart_config->enableRxRTS = false;
+		break;
+
+	case UART_CFG_FLOW_CTRL_RS485:
+		uart_config->enableTxCTS = false;
+		uart_config->enableRxRTS = false;
+		break;
```

Comment:
since these are the same you can use case fallthrough to eliminate a couple lines

Reply:
Good catch, @decsny .  Updated